### DISCUSSION TITLE
Can this library add a OpenHarmony platform?

### DISCUSSION THREAD

**Discussion by zt200655** [mono/SkiaSharp#2988] [Q&A]
How can this library add a build project for the OpenHarmony platform, or when will it be supported?

**Comment by mattleibow** [comment]
We could, but the best way to start would be to have a PR or issue with a docker file to build native skia.

Does the normal debian or musl build not work? Is you try build skia on a local machine, do you have errors building?

**Reply by zt200655** [reply]
When you attempt to build locally and encounter an error with git-sync-deps, here's a general translation of what you might see or describe:
![image](https://github.com/user-attachments/assets/4010e94a-d909-4324-822b-b46d6eef576d)

**Reply by mattleibow** [reply]
Did you check out submodules recursively?
